docroot/themes/custom/mappy/mappy.theme

```diff
@@ -4,3 +4,13 @@
  * @file
  * Functions to support theming in the Mappy theme.
  */
+
+function mappy_page_attachments_alter(&$page) {
```

Comment:
@AnneTee can you add a docblock for this? Is `phpcs --standard=Drupal` set up on your machine?